File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i=2
        j=2
        if(len(nums)<=2):
            return len(nums)

        while(j<len(nums)):
            if(nums[j] != nums[i-2]):
                nums[i] = nums[j]
                i += 1

            j+=1
        return i
        

File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 2;
        int j=2;
        int n = nums.size();
        if(n<=2){
            return n;
        }
        while(j<n){
            if(nums[j] != nums[i-2]){
                nums[i] = nums[j];
                i++;
            }
            j++;
        }
        return i;
        
    }
};